resources/js/Pages/Adminland/Hardware/Create.vue

```diff
@@ -153,14 +151,10 @@ export default {
   },
 
   methods: {
-    updateStatus(payload) {
-      this.form.lend_hardware = payload;
-    },
-
     submit() {
       this.loadingState = 'loading';
-      if (this.form.employee_id) {
-        this.form.employee_id = this.form.employee_id.value;
```

Comment:
@djaiss  `this.form.employee_id` now already contains ... the id, and only that (logical)